src/templates/Profile/Portfolio/AssetsTable/index.tsx

```diff
@@ -61,53 +63,107 @@ export const AssetsTable = ({ pools }: IAssetsTableProps) => {
         <S.Td>
           <S.ProductWrapper>
             <S.Imagecontainer>
-              <S.ImageWrapper>
-                {pool.logo ? (
-                  <img src={pool.logo} />
-                ) : (
-                  <Blockies size={8} scale={9} seed={pool.name ?? ''} />
-                )}
-              </S.ImageWrapper>
+              {pool?.id ? (
+                <S.ImageWrapper>
+                  {pool.logo ? (
+                    <img src={pool.logo} />
+                  ) : (
+                    <Blockies size={8} scale={9} seed={pool.name ?? ''} />
+                  )}
+                </S.ImageWrapper>
+              ) : (
+                <S.SkeletonLoadingWrapper>
+                  <SkeletonLoading
+                    height={2.4}
+                    width={2.4}
+                    borderRadios={999}
+                  />
+                </S.SkeletonLoadingWrapper>
+              )}
 
               <S.ChainLogoWrapper>
                 <Image src={pool.logoChain || comingSoonIcon} layout="fill" />
               </S.ChainLogoWrapper>
             </S.Imagecontainer>
             <S.FundWrapper>
-              <span>{pool.name}</span>
-              <span>{pool.symbol}</span>
+              {pool?.name ? (
+                <span>{pool.name}</span>
+              ) : (
+                <S.SkeletonLoadingWrapper>
+                  <SkeletonLoading height={1.8} width={14} />
+                </S.SkeletonLoadingWrapper>
+              )}
+              {pool?.symbol ? (
+                <span>{pool.symbol}</span>
+              ) : (
+                <S.SkeletonLoadingWrapper>
+                  <SkeletonLoading height={1.8} width={14} />
+                </S.SkeletonLoadingWrapper>
+              )}
             </S.FundWrapper>
           </S.ProductWrapper>
         </S.Td>
-        <S.Td>${parseFloat(pool.price).toFixed(2)}</S.Td>
         <S.Td>
-          ${pool.tvl ? BNtoDecimal(Big(pool?.tvl ?? Big(0)), 2) : '0'}
+          {pool?.price ? (
+            <>${parseFloat(pool.price).toFixed(2)}</>
+          ) : (
+            <S.SkeletonLoadingWrapper>
+              <SkeletonLoading height={1.8} width={8} />
+            </S.SkeletonLoadingWrapper>
+          )}
+        </S.Td>
+        <S.Td>
+          {pool?.tvl ? (
+            <>${pool.tvl ? BNtoDecimal(Big(pool?.tvl ?? Big(0)), 2) : '0'}</>
+          ) : (
+            <S.SkeletonLoadingWrapper>
+              <SkeletonLoading height={1.8} width={8} />
+            </S.SkeletonLoadingWrapper>
+          )}
         </S.Td>
         <S.Td>
-          <S.Change change={parseFloat(pool.changeMonth)}>
-            {pool.changeMonth}%
-          </S.Change>
+          {pool?.changeMonth ? (
```

Comment:
`changeMonth` could be zero

```suggestion
          {pool?.changeMonth != null ? (
```